### PR TITLE
USHIFT-622: Update rebase doc to reflect latest changes

### PR DESCRIPTION
**Pull request by fzdarsky**
Signed-off-by: Frank A. Zdarsky <fzdarsky@redhat.com>

Closes USHIFT-622

### CODE REVIEW

File: docs/rebase.md [hunk 1]
```diff
@@ -20,13 +21,23 @@ The following describes the current rebase process in more detail.
 
 On the machine used for the rebase,
 
-* install `git`, `golang` (>= 1.17), `oc`, `yq`, and `jq`,
-* add a pull secret into `~/.pull-secret.json`, and
+* install `git`, `golang` (>= 1.18), `oc`, `yq`, and `jq`,
```
Comment (ggiguash) [inline docs/rebase.md:24]: I think we should require the rebase to be done on a development machine and reference the docs here. 
configure-vm.sh should contain the code for downloading the right `yq`

Comment (fzdarsky) [reply docs/rebase.md]: Good idea. Done.

File: scripts/devenv-builder/configure-vm.sh [hunk 2]
```diff
@@ -47,6 +47,17 @@ sudo dnf update -y
 sudo dnf install -y git cockpit make golang selinux-policy-devel rpm-build bash-completion
 sudo systemctl enable --now cockpit.socket
 
+YQ_URL=https://github.com/mikefarah/yq/releases/download/v4.26.1/yq_linux_$(go env GOARCH)
+YQ_HASH_amd64=9e35b817e7cdc358c1fcd8498f3872db169c3303b61645cc1faf972990f37582
+YQ_HASH_arm64=8966f9698a9bc321eae6745ffc5129b5e1b509017d3f710ee0eccec4f5568766
```
Comment (mangelajo) [inline scripts/devenv-builder/configure-vm.sh:52]: 👍 good practice

**Comment by mangelajo** [comment]
/approve

I see @ggiguash comments were handled and the change looks good to me.

**Pull request merged**